### ISSUE TITLE
Tests for PR #530

### ISSUE THREAD

**Issue by chasingkirkjufell** [navcoin/navcoin-core#619]
This is a work in progress. 

Tests:
- [x] General checks [#25](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-539555670)
- [x] Overall QT visual check [#8](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-537525684) [#18](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-539211514) [#20](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-539211514) [#31](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-544634901) [#35](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-544683801) [#38](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-546609958)
- [x] Create consultation without answers
    - [x] Check fee amount
    - [x] Propose additional answers
        - [x] Propose several answers
        - [x] Propose duplicated answers [#44](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-547454507) [#49](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-548405126)
        - [x] Check fee amount
    - [x] Randomize question characters and answer characters to include special characters like `",-
    - [x] Try to vote during support phase
    - [x] Try to vote during reflection phase
    - [x] Check if supported according to consensus parameter
    - [x] Check support accepted only after passing minimum required set in consensus parameter
    - [x] Check reflection phase block cycle matches consensus parameter
    - [x] Cast vote
    - [x] Switch vote
    - [x] Cancel vote
    - [x] Abstain vote
    - [x] Check if voting phase block cycle count matches consensus parameter
    - [x] Check all support and vote count at all phases including ended consultations
    - [x] Review "View Chart" Window
    - [x] Check all the above in another node in the network
- [ ] Create consultation with answers [#2](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-537525684) [#4](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-537525684) [#10 ](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-538496979)[#15](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-539211514) [#26](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-540624710) [#28](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-544628680) [#29](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-544632562)
    - [x] Max 1 vote 
        - [x] Check fee amount
        - [x] Propose additional answers [#1](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-537525684) [#12](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-538547342)
            - [x] Propose several answers
            - [x] Check fee amount
        - [x] Randomize question characters and answer characters to include special characters like `",-
        - [x] Try to vote during support phase
        - [x] Try to vote during reflection phase
        - [x] Check if supported according to consensus parameter
        - [x] Check support accepted only after passing minimum required set in consensus parameter
        - [x] Check reflection phase block cycle matches consensus parameter
        - [x] Cast vote [#37](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-544734785)
        - [x] Switch vote
        - [x] Cancel vote
        - [x] Abstain vote
        - [x] Check if voting phase block cycle count matches consensus parameter [#17](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-539211514)
        - [x] Check all support and vote count at all phases including ended consultations [#21](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-539211514) 
        - [x] Review "View Chart" Window at all phases [#5](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-537525684) [#13](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-539211514) [#35-1](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-544683801)
        - [x] Check all the above in another node in the network [#45](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-547502693)
    - [ ] Max more than 1 vote
        - [x] Check fee amount
        - [x] Propose additional answers
            - [x] Propose several answers
            - [x] Check fee amount
        - [x] Randomize question characters and answer characters to include special characters like `",-
        - [x] Try to vote during support phase
        - [ ] Try to vote during reflection phase
        - [x] Check if supported according to consensus parameter
        - [x] Check support accepted only after passing minimum required set in consensus parameter
        - [x] Check reflection phase block cycle matches consensus parameter
        - [x] Cast vote
        - [x] Switch vote
        - [x] Cancel vote
        - [x] Abstain vote
        - [x] Check if voting phase block cycle count matches consensus parameter
        - [x] Check all support and vote count at all phases including ended consultations [#22](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-539211514) 
        - [x] Review "View Chart" Window at all phases
        - [ ] Check all the above in another node in the network

- [ ] Create range consultation [#6](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-537525684) [#33](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-544643784)
    - [x] Randomize question characters and answer characters to include special characters like `",-
    - [ ] Try to vote during support phase
    - [ ] Try to vote during reflection phase
    - [x] Check if supported according to consensus parameter [#34](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-544682739)
    - [x] Check support accepted only after passing minimum required set in consensus parameter
    - [x] Check reflection phase block cycle matches consensus parameter
    - [ ] Cast vote [#42](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-546962830)
    - [x] Switch vote
    - [x] Cancel vote
    - [x] Abstain vote [#40](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-546609958)
    - [x] Check if voting phases block cycle matches consensus parameter
    - [x] Check all support and vote count at all phases including ended consultations
    - [x] Review "View Chart" Window [#39](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-546609958) [#41](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-546609958)
    - [ ] Check all the above in another node in the network

- [ ] Change Length in blocks of a voting cycle
    - [x] Propose change (test non positive number including 0) [#7](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-537525684) [#19](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-539211514) 
    - [x] Vote
    - [x] Check accepted
    - [x] Check behavior change [#36](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-544703337)
    - [x] Redo consultation tests after parameter change
    - [ ] Check all the above in another node in the network [#46](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-547934031)
- [ ] Minimum of support needed for starting a consultation 
    - [x] Propose change (test non positive number including 0)
    - [x] Vote
    - [x] Check accepted
    - [x] Check behavior change
    - [x] Redo consultation tests after parameter change
    - [ ] Check all the above in another node in the network
- [ ] Earliest cycle when a consultation can get in confirmation phase [#27](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-544624769)
    - [x] Propose change (test non positive number including 0)
    - [x] Vote
    - [x] Check accepted
    - [x] Check behavior change
    - [x] Redo consultation tests after parameter change
    - [ ] Check all the above in another node in the network
- [x] Length in cycles for consultation votings
    - [x] Propose change (test non positive number including 0)
    - [x] Vote
    - [x] Check accepted
    - [x] Check behavior change
    - [x] Redo consultation tests after parameter change
    - [x] Check all the above in another node in the network
    - [x] Check the behavior of consultations that have gone passed the new proposed length once a change is accepted

- [ ] Maximum of voting cycles for consultation to gain support [#52](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-548448319)
    - [x] Propose change (test non positive number including 0)
    - [x] Vote [#43](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-547101634)
    - [x] Check accepted
    - [x] Check behavior change
    - [x] Redo consultation tests after parameter change
    - [x] Check all the above in another node in the network
    - [x] Check the behavior of consultations that have already started before a parameter change and their behavior with the new parameter.
- [ ] Length in cycles for the reflection phase of consultations
    - [x] Propose change (test non positive number including 0)
    - [x] Vote
    - [x] Check accepted
    - [x] Check behavior change [#53](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-548448319)
    - [ ] Redo consultation tests after parameter change
    - [ ] Check all the above in another node in the network
- [ ] Minimum fee to submit a consultation
    - [x] Propose change (test non positive number including 0)
    - [x] Vote
    - [x] Check accepted
    - [x] Check behavior change
    - [x] Redo consultation tests after parameter change
    - [ ] Check all the above in another node in the network
- [ ] Minimum of support needed for a consultation answer proposal
    - [x] Propose change (test non positive number including 0)
    - [x] Vote
    - [x] Check accepted
    - [x] Check behavior change
    - [x] Redo consultation tests after parameter change
    - [ ] Check all the above in another node in the network
- [ ] Minimum fee to submit a consultation answer proposal
    - [x] Propose change (test non positive number including 0)
    - [x] Vote
    - [x] Check accepted
    - [x] Check behavior change
    - [x] Redo consultation tests after parameter change
    - [ ] Check all the above in another node in the network
- [ ] 	Minimum of quorum for fund proposal votings
    - [x] Propose change (test non positive number including 0)
    - [x] Vote
    - [x] Check accepted
    - [x] Check behavior change
    - [ ] Check all the above in another node in the network
- [ ] 	Minimum of positive votes for a fund proposal to be accepted
    - [x] Propose change (test non positive number including 0)
    - [x] Vote
    - [x] Check accepted
    - [x] Check behavior change
    - [ ] Check all the above in another node in the network
- [ ] 	Minimum of negative votes for a fund proposal to be rejected
    - [x] Propose change (test non positive number including 0)
    - [x] Vote
    - [x] Check accepted
    - [x] Check behavior change
    - [x] Check all the above in another node in the network
- [ ] 	Minimum fee to submit a fund proposal
    - [x] Propose change (test non positive number including 0)
    - [x] Vote
    - [x] Check accepted
    - [x] Check behavior change
    - [ ] Check all the above in another node in the network
- [ ] 	Maximum of voting cycles for fund proposal votings
    - [x] Propose change (test non positive number including 0)
    - [x] Vote
    - [x] Check accepted
    - [x] Check behavior change
    - [ ] Check all the above in another node in the network
- [ ] 	Minimum of quorum for payment request votings
    - [x] Propose change (test non positive number including 0)
    - [x] Vote
    - [x] Check accepted
    - [x] Check behavior change
    - [ ] Check all the above in another node in the network
- [ ] 	Minimum of positive votes for a payment request to be accepted
    - [x] Propose change (test non positive number including 0)
    - [x] Vote
    - [x] Check accepted
    - [x] Check behavior change
    - [ ] Check all the above in another node in the network
- [ ] 	Minimum of negative votes for a payment request to be rejected
    - [x] Propose change (test non positive number including 0)
    - [x] Vote
    - [x] Check accepted
    - [x] Check behavior change
    - [ ] Check all the above in another node in the network
- [ ] 	Minimum fee to submit a payment request
    - [x] Propose change (test non positive number including 0)
    - [x] Vote
    - [x] Check accepted
    - [ ] Check behavior change [#63](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-549947080)
    - [ ] Check all the above in another node in the network
- [ ] 	Maximum of voting cycles for fund proposal votings
    - [x] Propose change (test non positive number including 0)
    - [x] Vote
    - [x] Check accepted
    - [x] Check behavior change
    - [ ] Check all the above in another node in the network
- [ ] 	Frequency of the fund accumulation transaction
    - [x] Propose change (test non positive number including 0)
    - [x] Vote
    - [x] Check accepted
    - [x] Check behavior change
    - [ ] Check all the above in another node in the network
- [x] 	Percentage of generated NAV going to the Fund
    - [x] Propose change (test non positive number including 0) [#32](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-544640058) 
    - [x] Vote
    - [x] Check accepted
    - [x] Check behavior change
    - [x] Check all the above in another node in the network
- [x] 	Amount of NAV generated per block 
    - [x] Propose change (test non positive number including 0) [#30](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-544633404) 
    - [x] Vote [#24](https://github.com/navcoin/navcoin-core/pull/530#issuecomment-539311576) 
    - [x] Check accepted
    - [x] Check behavior change
    - [x] Check all the above in another node in the network
- [ ] 	Yearly fee of registering a name in NavNS
    - [ ] Propose change (test non positive number including 0)
    - [ ] Vote
    - [ ] Check accepted
    - [ ] Check behavior change
    - [ ] Check all the above in another node in the network
- [ ] 	Minimum fee as a fund contribution to submit a DAO vote using a light wallet
    - [ ] Propose change (test non positive number including 0)
    - [ ] Vote
    - [ ] Check accepted
    - [ ] Check behavior change
    - [ ] Check all the above in another node in the network

**Comment by mxaddict** [comment]
@chasingkirkjufell I think we can close this, since we've already merged into master ❤️
